src/V3EmitC.cpp

```diff
@@ -1385,8 +1385,19 @@ class EmitCStmts VL_NOT_FINAL : public EmitCBaseVisitor {
         AstNodeDType* dtypep = nodep->varp()->dtypep();
         bool useDly = useDelayedValue(nodep);
         if (useDly) puts("verilated_nba_ctrl.get_scheduled(&");
+        if(nodep->abovep() != nullptr)
+            if(nodep->isWide() and nodep->abovep()->type() == AstType::atIf)
```

Comment:
Please use the `VN_IS` macro for checking AST types (`VN_IS(nodep, If)`)

Reply:
thanks for this remark